### DISCUSSION TITLE
`WARNING:cflib.crtp.usbdriver:Exception while scanning for Crazyflie USB: No backend available`

### DISCUSSION THREAD

**Discussion by r-bt** [bitcraze/discussions#1333] [Q&A]
Environment:

- macOS 14.4 
- Python 3.11.7 & 3.12.0

I fresh installed Python 3.11.7 and cfclient and I connect both a Crazyflie Radio PA and a microUSB cable connected to a Crazyflie. However when I get the following error:

```
WARNING:cflib.crtp.usbdriver:Exception while scanning for Crazyflie USB: No backend available
```

**Comment by ToveRumar** [comment]
Hi!

Is this issue present when using python 3.11.7 as well? 
We have had this issue with macOS and python 3.12 before and we have an issue for it [here](https://github.com/bitcraze/crazyflie-lib-python/issues/446) which you could follow for updates on this issue. 

I'm interpreting from your other tickets that this is working with python 3.11.7. Am I misunderstanding?
